app/uk/gov/hmrc/economiccrimelevyregistration/models/grs/IncorporatedEntityJourneyData.scala

```diff
@@ -36,7 +34,7 @@ object IncorporatedEntityJourneyData {
 final case class CompanyProfile(
   companyName: String,
   companyNumber: String,
-  dateOfIncorporation: LocalDate,
+  dateOfIncorporation: String,
```

Comment:
In case you wonder why I've turned this from a `LocalDate` to a `String` - it's because this is the type they use in the GRS service, and this is because the companies house data is shoddy in that if there is no date of incorporation, they return an empty string - so `LocalDate` would fail to deserialize (you could represent it as an `Option[LocalDate]` and have a custom writer but I felt that was overkill for this as we just store / audit this data.